content/post/kubedb-v2024.08.16/index.md

```diff
@@ -0,0 +1,1689 @@
+---
+title: Announcing KubeDB v2024.08.16
+date: ""
```

Comment:
Add valid date
Format "2024-08-19"

Reply:
Vai this will be adjusted once release date is fixed.